diagnostics-app/src/config/config.ts

```diff
@@ -25,6 +25,24 @@ export const VISIBLE_TELEMETRY_CARDS: TelemetryInfoType[] = [
   TelemetryInfoType.TPM,
 ];
 
+export const VISIBLE_EVENT_CARDS: EventCategory[] = [
```

Comment:
Could you mention this in the commit message?

Maybe `Move visible event cards to customization config`

Reply:
Done.